packages/cosmic-swingset/lib/ag-solo/vats/bootstrap.js

```diff
@@ -69,16 +76,18 @@ export default function setup(syscall, state, helpers) {
         const chainTimerService = await E(vats.timer).createTimerService(
           timerDevice,
         );
-        const zoe = await E(vats.zoe).getZoe();
-        const contractHost = await E(vats.host).makeHost();
 
         // Make the other demo mints
         const issuerNames = ['moola', 'simolean'];
-        const issuers = await Promise.all(
-          issuerNames.map(issuerName =>
-            E(vats.mints).makeMintAndIssuer(issuerName),
+        const [zoe, contractHost, issuers] = await Promise.all([
```

Comment:
why is this better? it's not like we have any meaningful parallelism within the kernel, and everything here is local, and it requires the reader to remember/compare the order of items in `[zoe, contractHost, issuers]` against the order of the inputs to `Promise.all`, whereas the original did exactly one thing per line. And this is unrelated to the networking changes.

Reply:
I'll revert.  I was doing something more in that change before.